resources/help/spellcheck.md

```diff
@@ -0,0 +1,35 @@
+# Spellcheck
+
+This module offers methods to perform spellchecking using a Hunspell 
+compatible dictionary. Before use, the spellcheck module must be
+initialized by providing paths to an AFF file, and a dictionary file.
+
```

Comment:
A minor extension of this intro that makes it clear that this module is **not** a fully fledged spellchecker.
Feel free to promote your own plugin here :wink: 

Something like:

> This module is not a complete spellchecking solution. It provides methods for checking the spelling of a word and finding spelling suggestions for a particular word for use within scripts.
> For a more complete spellchecker there are plugins available. Such as:
>
> - Your solution with link